common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -61,6 +61,35 @@ const KeyTypesOfKeyMap = {
 // based on available hints.  (i.e., `layout.defaultHint == 'flick'`)
 const KeyTypesOfFlickList = ['n', 'ne', 'e', 'se', 's', 'sw', 'w', 'nw'] as const;
 
+/**
+ * Copies non-computed properties and enumerable scomputed property definitions
+ * for any properties not yet defined on the target object.
+ * @param rawObj
+ * @param defaults
+ */
+function assignDefaultsWithPropDefs<RawType, Type extends RawType>(rawObj: RawType, defaults: Type) {
```

Comment:
Why `raw` here? What does 'raw' mean?

Reply:
raw:  has the same fields, but none of the methods or defined properties.

The idea is that we have a deserialized-from-JSON version of a class instance, but it's not actually of that type because it lacks those things.  We put them in place so the "deserialized" object essentially becomes a class instance.

Of course, the 'raw' instance won't have the underlying `prototype`, but so long as that's the only difference, we're good - we don't do prototype-based checks against keyboard-layout info.

Reply:
Feel free to suggest alternate terminology if you want; 'raw' is what made sense to me in the moment.